reactor/ip_addr_changed.sls

```diff
@@ -0,0 +1,27 @@
+{%- set api_servers = salt['pillar.get']('roles:kube-master', '') %}
+{%- set sender      = data['id'] %}
+
+{% if sender in api_servers %}
```

Comment:
We should trigger the `hosts-master` in this case as well, as we may have a kubelet on the masters that need to have the api address. In #44 I made the `hosts-master` role render the API address for this reason.